### ISSUE TITLE
Accessibility issues in Playwright tests on the consumer side

### ISSUE THREAD

**Issue by spli02** [fortanix/baklava#531]
When running the accessibility Playwright tests on the consumer side, the following issues were found from baklava components. These should be fixed to comply accessibility rules.

1. `MenuProvider` component (Issue: Invalid ARIA attribute)
```
Rule: aria-allowed-attr
Help: Elements must only use supported ARIA attributes
Help URL: https://dequeuniversity.com/rules/axe/4.10/aria-allowed-attr?application=playwright
Affected nodes:
  - Target: span[aria-controls="listbox-«r1c»"]
    Fix all of the following:
  ARIA attribute is not allowed: aria-expanded="false"
```
________

2. `MultiSearch` component (Issue: Nested interactive elements and target size)
```
Rule: nested-interactive
Help: Interactive controls must not be nested
Help URL: https://dequeuniversity.com/rules/axe/4.10/nested-interactive?application=playwright
Affected nodes:
  - Target: ._8U3gIq_bk-search-input
    Fix any of the following:
  Element has focusable descendants
```

```
Rule: target-size
Help: All touch targets must be 24px large, or leave sufficient space
Help URL: https://dequeuniversity.com/rules/axe/4.10/target-size?application=playwright
Affected nodes:
  - Target: ._8U3gIq_bk-search-input
    Fix any of the following:
  Target has insufficient size (350px by 23.4px, should be at least 24px by 24px)
  Target has insufficient space to its closest neighbors. Safe clickable space has a diameter of 0px instead of at least 24px.
  - Target: input[placeholder="Search"]
    Fix any of the following:
  Target has insufficient size (328px by 22.4px, should be at least 24px by 24px)
  Target has insufficient space to its closest neighbors. Safe clickable space has a diameter of 0px instead of at least 24px.

```
________
3. `Tab` component (Issue: Invalid role structure) 
```
Rule: aria-required-children
Help: Certain ARIA roles must contain particular children
Help URL: https://dequeuniversity.com/rules/axe/4.10/aria-required-children?application=playwright
Affected nodes:
  - Target: .S904AW_bk-tabs
    Fix any of the following:
  Element has children which are not allowed: ul[tabindex], search[tabindex], button[aria-controls], [role=table], button[aria-disabled]

```
```
Rule: aria-required-parent
Help: Certain ARIA roles must be contained by particular parents
Help URL: https://dequeuniversity.com/rules/axe/4.10/aria-required-parent?application=playwright
Affected nodes:
  - Target: li[data-tab="files"]
    Fix any of the following:
  Required ARIA parent role not present: tablist
  - Target: li[data-tab="data_sources"]
    Fix any of the following:
  Required ARIA parent role not present: tablist
```
```
Rule: list
Help: <ul> and <ol> must only directly contain <li>, <script> or <template> elements
Help URL: https://dequeuniversity.com/rules/axe/4.10/list?application=playwright
Affected nodes:
  - Target: ul
    Fix all of the following:
  List element has direct children that are not allowed: [role=tab]

```
